src/Microsoft.AspNetCore.AzureKeyVault.HostingStartup/Microsoft.AspNetCore.AzureKeyVault.HostingStartup.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+  <Import Project="..\..\build\hostingstartup.targets" />
+
+  <PropertyGroup>
+    <Description>ASP.NET Core lightup integration with Azure KeyVault.</Description>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
Not netstandard? Or is this for deps generation?

Reply:
Why are the other hosting startup targetting netcoreapp2.0 and net461 then? https://github.com/aspnet/AzureIntegration/blob/release/2.0/src/Microsoft.AspNetCore.AzureAppServices.HostingStartup/Microsoft.AspNetCore.AzureAppServices.HostingStartup.csproj#L8